### DISCUSSION TITLE
Allow whitespace character in prefixes for Autolink references

### DISCUSSION THREAD

**Discussion by csadilek** [community/community#37556] [Other Feature Feedback, Questions, & Ideas]
Hi,

I was trying to create an autolink reference and match `Bug 1234567 - description`, but the space character it not allowed in  prefixes: `Reference prefix must only contain letters, numbers, or .-_+=:/# `

This could be worked around using a different separator, of course, but would require breaking away from our standard commit message format. Specifically, I am trying to link commits to Bugzilla tickets.

It would be great if this was possible somehow. In https://github.com/community/community/discussions/4725 support for `.` was added, https://github.com/community/community/discussions/29565 seems more flexible than I'd need.

:)

Thank you for considering!

**Comment by ossanna16** [comment]
Hi there @csadilek and welcome to our community! Thank you for the feedback 🙂

**Comment by kyle-blair** [comment]
I would add that allowing a whitespace character would mean autolinks worked for standard git trailer format. Git commands enforce a space in between the key and value for trailers, meaning I can't autolink trailers. For example:
`$ git commit --message "feat: speed up page load" --trailer "key:value"`
This adds a standard trailer to my commit message even though I provided no space character in my "key:value" argument, git added one:
```shell
$ git log -1 --format=%B
feat: speed up page load

key: value
```
See the [git trailer documentation](https://git-scm.com/docs/git-interpret-trailers).
